CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ endif()
 
 find_package(Doxygen)
 if(Doxygen_FOUND)
+  set(DOXYGEN_GENERATE_XML YES) 
```

Comment:
So ultimately coverage with Doxygen requires coverage to assess the doxygen-generated XML files?

Reply:
Yes, it needs the xml outputs!